src/components/App/App.jsx

```diff
@@ -1,6 +1,7 @@
 import './App.css';
 import Display from '../Display/display.component';
 import ButtonPanel from '../ButtonPanel/button-panel.component';
+import '../logic/calculate';
```

Comment:
Change your import statement to `import calculate from '../logic/calculate';` to get proper access to the calculate function.